_data/concepts-landing-page.yml

```diff
@@ -28,10 +28,10 @@ active_tab:
         description: How we display financial assistance data
         total_concepts: 1
         link: /concepts/cfda-program-summary
-       - headline:CFDA Program Summary
+       - headline: CFDA Program Summary
```

Comment:
Thank you. Keeping Jekyll happy is a full time job.

From: Austin Valeske [mailto:notifications@github.com]
Sent: Wednesday, July 27, 2016 3:27 AM
To: fedspendingtransparency/openbeta
Cc: Siempelkamp, Alice; Author
Subject: Re: [fedspendingtransparency/openbeta] Concepts Landing Page Update (#46)


In _data/concepts-landing-page.yml<https://github.com/fedspendingtransparency/openbeta/pull/46#discussion_r72391828>:

> @@ -28,10 +28,10 @@ active_tab:

>          description: How we display financial assistance data

>          total_concepts: 1

>          link: /concepts/cfda-program-summary

> -       - headline:CFDA Program Summary

> +       - headline: CFDA Program Summary

This line was indented an extra space, so jekyll got mad. I fixed it though. :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/fedspendingtransparency/openbeta/pull/46/files/f12213fe229129a79b1470818acd4544ca1c19bf#r72391828>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATHnP1_ILrMgjpwFAD6uJD3hOWW7WMIKks5qZwhVgaJpZM4JVjxn>.